fossilize_external_replayer_linux.hpp

```diff
@@ -49,10 +49,60 @@
 #include "platform/gcc_clang_spinlock.hpp"
 #endif
 
+#ifdef __linux__
```

Comment:
The idea makes sense, is there any documentation I can refer to here?

Reply:
For SCHED_BATCH: https://www.kernel.org/doc/html/latest/scheduler/sched-design-CFS.html

For the ioprio stuff, I'm going to add the relevant link.

Reply:
Resolved?

Reply:
Moved to #105